src/OpenSearch.Client/Requests.Cluster.cs

```diff
@@ -340,12 +367,21 @@ public partial class ClusterPutSettingsRequest : PlainRequestBase<ClusterPutSett
 		}
 
 		///<summary>Explicit operation timeout for connection to master node</summary>
+		///<remarks>Deprecated as of OpenSearch 2.0, use <see cref="ClusterManagerTimeout"/> instead</remarks>
 		public Time MasterTimeout
```

Comment:
Should this call `ClusterManagerTimeout`?

Reply:
As discussed above - no changes required.